test/edu/ucsb/nceas/metacattest/EventLogIT.java

```diff
@@ -123,8 +137,9 @@ public void testIsDeleted() throws Exception{
      * Test logging when the feature is disabled
      * @throws Exception
      */
+    @Test
     public void testDisableEventLog() throws Exception {
-        PropertyService.setPropertyNoPersist("event.log.disabled", "true");
+        PropertyService.setPropertyNoPersist("event.log.enabled", "false");
```

Comment:
instead of modifying properties, let's try to use this pattern:

~~PropertyService.setPropertyNoPersist("event.log.enabled", "false");~~

```java
  Properties withProperties = new Properties();
  withProperties.setProperty("event.log.enabled", "false");
  try (MockedStatic<PropertyService> ignored =
          LeanTestUtils.initializeMockPropertyService(withProperties)) {

    // ...test code here
  }
```